src/array_string.rs

```diff
@@ -295,6 +295,36 @@ impl<const CAP: usize> ArrayString<CAP>
         Ok(())
     }
 
+    pub fn try_push_iterator<I: IntoIterator<Item = char>>(&mut self, value: I) -> Result<(), CapacityError<()>> {
```

Comment:
```suggestion
    pub fn try_extend<I: IntoIterator<Item = char>>(&mut self, value: I) -> Result<(), CapacityError<()>> {
```
I think the non-failing method is called `extend`: [`Vec::extend`](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.extend).